### ISSUE TITLE
BALLOONING DEGENERATION

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6241]
**Preferred term label:**
hepatocyte ballooning

**Synonyms**
 ballooning degerenation of hepatocytes

**Definition (free text, please give PubMed ID)**
Evidence of hepatocyte injury in the form of ballooning is required for diagnosis of steatohepatitis. The term ballooning corresponds to swelling of the hepatocyte, rounding of its contour, and alteration of the cytoplasm, which takes on a reticulated, rarified, or flocculant quality. The cytoplasm of the ballooned hepatocytes often contains clumps of eosinophilic ropey material known as Mallory-Denk bodies (MDBs) or Mallory hyaline, which is composed of hyperphosphorylated misfolded intermediate filaments, ubiquitin, and ubiquitin-binding protein P62.
PMID: 31603713
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Comment--  When evaluating ballooning, it is best to scan the slide at ×40 to ×100 (×4 and ×10 objectives) looking for enlarged hepatocytes with rarified or flocculent cytoplasm and to confirm at ×200 (×20 objective) to avoid overcalling. Ballooned hepatocytes should be the size of large droplets of macrovesicular steatosis in adjacent hepatocytes and can be larger (Figure 3, A and B). Early in the course of the disease, it is best to look for ballooned hepatocytes in zone 3 (around the central vein) because they tend to occur there, but as the disease progresses they can be found in the lobules irrespective of zonal distribution. The trichrome stain can be helpful in identifying ballooned hepatocytes, particularly when focusing on the central vein in areas of pericellular fibrosis

**Comment by pnrobinson** [comment]
A process associated with degenerating hepatocytes where they swell in size up to twofold or threefold with a ‘wispy’ clear cytoplasm often containing Mallory-Denk bodies; associated with steatohepatitis and cholate stasis  Mallory-Denk  bodyInclusions found within the cytoplasm of ballooned hepatocytes. Composed of cytokeratin filaments, they have a ‘twisted-rope’ appearance  PMID: 31885839

**Comment by pnrobinson** [comment]
done
